packages/extension/src/background/handlers/State.ts

```diff
@@ -91,7 +91,7 @@ export default class State {
     extension.windows.create({
       // This is not allowed on FF, only on Chrome - disable completely
       // focused: true,
-      height: 580,
+      height: 581,
```

Comment:
yes, well...

Reply:
The window size doesn't need to remain this ugly, but it seems like it needs to remain > body/html size.
I tried many things like setting the style in the `html` with the body `100%`, with nothing in the body but that vertical scrollbar remains.

Reply:
I understand - the basic issue is that the borders are included in the width/height when making a popup. Just for interest, did you try `box-sizing`?

Happy as it is, since it, well, is what it is :)

Reply:
I tried just to remove box-sizing which broke the sizing altogether iirc, but didn't play further with it. Will do a last chance play with box-sizing tomorrow.

Reply:
No, it is fine. Was just interested. (i.e. basically had issues like this ages ago, the work-around was similar to what was actually done here - so was just wondering if there are maybe better options. Would not spend too much additional effort)